capstone/src/components/User/Experience/Performance/Comment.jsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Badge, Text } from '@chakra-ui/react';
+
+const data = [
+  { score: 1, comment: ['really bad food'] },
+  { score: 2, comment: ['not that bad', 'ok if you are really hungry', 'need more work'] },
+  { score: 3, comment: ['can improve but ok'] },
+  { score: 4, comment: ['it was ok'] },
+  { score: 5, comment: ['my favorite place', 'it changed my life'] },
+];
+
+export default function Comment({determineColor}) {
```

Comment:
It looks like we're rendering multiple comments. Maybe we should name this `CommentArea` or `CommentSection` to be clearer?

Reply:
Definitely! I will add it in the upcoming commit